news/2019-02-04-community-choice-2018-results.md

```diff
@@ -0,0 +1,143 @@
+
+[Vickeblanka - Black Rover (TV Size) by Sotarks](https://osu.ppy.sh/beatmapsets/781509) with **72** votes total.
+
+[ESTi - HELIX (Edit ver.) by hikikochan](https://osu.ppy.sh/beatmapsets/625729) with **71** votes total.
+
+[ClariS - SHIORI vs. Hitorigoto by Sotarks](https://osu.ppy.sh/beatmapsets/707380) with **67** votes total.
+
+[XXXTENTACION - Jocelyn Flores by Chromoxx](https://osu.ppy.sh/beatmapsets/705133) with **66** votes total.
+
+[Will Stetson - Harumachi Clover (Swing Arrangement) [Dictate Edit] by Sotarks](https://osu.ppy.sh/beatmapsets/842412) with **63** votes total.
+
+[KASAI HARCORES - Cycle Hit by Axer](https://osu.ppy.sh/beatmapsets/668239) with **63** votes total.
+
+[Check out the full results over on the contest listing for this mode!](https://osu.ppy.sh/community/contests/75)
+
+### osu!catch
+
+[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/asphyxia.jpg)](https://osu.ppy.sh/beatmapsets/758101)
+
+With **182** votes, the collaborative sets by [Ascendance](https://osu.ppy.sh/users/2931883), [Spectator](https://osu.ppy.sh/users/702598) and [Hareimu](https://osu.ppy.sh/users/4138746) on [Co shu Nie - asphyxia (TV edit)](https://osu.ppy.sh/beatmapsets/758101) hosted by [Monstrata](https://osu.ppy.sh/users/2706438) strangled out the competition solidly with a nearly 50 point lead.
+
+Similar to osu!taiko, the voting for osu!catch was a super close endeavour all up until the last three or four days of voting, when asphyxia surged forward with a lead that the other sets could just not recover from.
+
+The agonizing choice many osu!catch players were forced to make is super obvious in the sheer vote gap between many of the finalist entries, shown purely in the fact that 2nd and 3rd place are literally one vote apart. 
+
+Yowza.
+
+The finalists were:
+
+[CustomiZ - COOLEST by Ascendance](https://osu.ppy.sh/beatmapsets/502512) with **139** votes total.
+
+[Camellia - Ultimate Ascension by Deif](https://osu.ppy.sh/beatmapsets/520301) with **138** votes total.
+
+[Chito (CV: Minase Inori), Yuuri (CV: Kubo Yurika) - More One Night (Assertive Hardcore Bootleg) \[short ver.\] by Ascendance](https://osu.ppy.sh/beatmapsets/781081) with **136** votes total.
+
+[Icon For Hire - Make a Move (Speed Up Ver.) by Sotarks](https://osu.ppy.sh/beatmapsets/765778) with **134** votes total.
+
+[LeaF - Alice in Misanthrope -Ensei Alice- by Deif](https://osu.ppy.sh/beatmapsets/797236) with **113** votes total.
+
+[Function Phantom - Algebra by Spectator](https://osu.ppy.sh/beatmapsets/468281) with **107** votes total.
+
+[dj TAKA - quaver by Ascendance](https://osu.ppy.sh/beatmapsets/879085) with **102** votes total.
+
+[LiSA - ADAMAS (TV Size) by Ascendance](https://osu.ppy.sh/beatmapsets/867287) with **100** votes total.
+
+[Zektbach - Turii \~Panta rhei\~ by Kyuare](https://osu.ppy.sh/beatmapsets/685428) with **99** votes total.
```

Comment:
panta rhei is still crossed out

Reply:
you must be looking at an old revision, it's corrected now